app/views/dashboard/ng-partials/widgets/_logs.html.haml

```diff
@@ -10,9 +10,9 @@
           %table
             %thead
               %tr
-                %th{width: '10%'}
+                %th{width: '15%'}
                   %label TIME
-                %th{width: '80%'}
+                %th{width: '75%'}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace